utils/utils_test.go

```diff
@@ -0,0 +1,48 @@
+package utils
+
+import (
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+const (
+	defaultPathEnv = "/host/opt/cni/bin"
+	defaultBoolEnv = false
+
+	envPath = "Path"
+	envBool = "Bool"
+)
+
+// Validate that GetBoolAsStringEnvVar runs against acceptable format input without error
+func TestGetBoolAsStringEnvVar(t *testing.T) {
+	// Test environmental flag variable not set
+	tmp, err := GetBoolAsStringEnvVar("Bool", defaultBoolEnv)
+	assert.NoError(t, err)
+	assert.Equal(t, tmp, defaultBoolEnv)
+
+	// Test basic Boolean as string set with acceptable format
+	_ = os.Setenv("Bool", "True")
+	tmp, err = GetBoolAsStringEnvVar(envBool, defaultBoolEnv)
+	assert.NoError(t, err)
+	assert.Equal(t, tmp, true)
+
+	// Test basic Boolean as string set with unacceptable format
+	_ = os.Setenv("Bool", "TrUe")
+	tmp, err = GetBoolAsStringEnvVar(envBool, defaultBoolEnv)
+	assert.Error(t, err)
+	assert.Equal(t, tmp, false)
+}
+
+// Validate that GetEnv runs without error against environmental variable with type other than boolean as string
+func TestGetEnv(t *testing.T) {
+	// Test environmental flag variable not set
+	tmp := GetEnv("Path", defaultPathEnv)
+	assert.Equal(t, tmp, defaultPathEnv)
+
+	// Test environmental flag variable set
+	_ = os.Setenv("Path", "/host/opt/cni/bin/test")
```

Comment:
As mentioned above, use the global variable you defined and unset the environment variable after the function runs